app/views/chapters/show.html.erb

```diff
@@ -3,24 +3,36 @@
 <% end %>
 
 <div class="chapter-description">
-  <h1>
-    <span class="hidden-xs"><%= t(:chapter_number, number: @chapter.number) %>:&nbsp;</span>
-    <span><%= @chapter.name %></span>
+  <h1 class="pull-left">
+    <span class="hidden-xs pull-left"><%= t(:chapter_number, number: @chapter.number) %>:&nbsp;</span>
+    <span class="pull-left"><%= @chapter.name %></span>
+    <% if @chapter.monolesson? %>
+      <div class='mu-monolesson pull-left'>
+        <%= render partial: 'layouts/guide_title_icons' %>
+      </div>
+    <% end %>
   </h1>
 
   <%= @chapter.description_html %>
 
 </div>
 
 <% if @chapter.lessons.present? %>
-<div>
-  <h3><%= t(:lessons) %></h3>
+  <% if @chapter.monolesson? %>
+    <div class='mu-monolesson'>
```

Comment:
Could we please use " instead of ' consistently?

Reply:
Done!